Interface/AddOns/nMainbar/config.lua

```diff
@@ -45,6 +45,12 @@ nMainbar.Config = {
         font = 'Fonts\\ARIALN.ttf',
     },
 
+    artifactBar = {
+        mouseover = true,
```

Comment:
The mouseover is just for the text just like the ReputationWatchBar. The settings could probably be merged with the repBar settings because they are doing the same thing.

Here is watch the Artifact Watch Bar looks like in game.

http://imgur.com/G2W4CzQ